froide/foirequest/urls/request_urls.py

```diff
@@ -203,4 +210,6 @@
     # Feed
     path("<slug:slug>/feed/", FoiRequestFeedAtom(), name="foirequest-feed_atom"),
     path("<slug:slug>/rss/", FoiRequestFeed(), name="foirequest-feed"),
+    # Shortlink for message
+    path("m/<int:obj_id>/", message_shortlink, name="foirequest-message_shortlink"),
```

Comment:
I see what you did there. It sure comes in handy. Our other more or less 'official' shortlinks (`/r/` and `/b/` etc.) are translated (in German to `/a/` and `/b/`), but we can keep it at a more low profile as well.

I would move this up above the `<slug:slug>` prefix to the other short link URL for some order and to match before slug (unlikely) matches the 'm' prefix).